samples/bluetooth/st_ble_sensor/src/led_svc.c

```diff
@@ -29,19 +33,30 @@ bool led_state;
 void led_on_off(u32_t led_state)
 {
 	if (led_dev) {
-		gpio_pin_write(led_dev, LED, led_state);
+		gpio_pin_set(led_dev, DT_ALIAS_LED0_GPIOS_PIN, (int)led_state);
```

Comment:
Either use `LED` defined above or get rid of the local defines and use the devicetree symbol consistently.